user/enterprise/custom-queues.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Enterprise Customer Worker Queues
+layout: en_enterprise
+---
+
+Setting up custom worker queues on Travis CI Enterprise allows your team to designate a particular worker (or group of workers) for building specific repositories. This is especially useful for differentiating workers that have been customized specially.
+
```

Comment:
that is a lot of different ways of saying 'different', but doesn't actually say much :-p

Reply:
BUT EVERYTHING IS SO DIFFERENT

Reply:
thanks for catching that - will fix :)

Reply:
We're all special snowflakes here.